Mathlib/SetTheory/Descriptive/Tree.lean

```diff
@@ -0,0 +1,31 @@
+/-
+Copyright (c) 2024 Sven Manthe. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sven Manthe
+-/
+import Mathlib.Order.CompleteLattice.SetLike
+
+/-!
+# Trees in the sense of descriptive set theory
+
+This file defines trees of depth ω in the sense of descriptive set theory.
```

Comment:
I haven't seen trees in the sense of descriptive set theory. Of course, you don't have to explain all definitions in prose --- but if you can, it might help. Can you give a short summary what these are? If so, that seems useful to include.

Reply:
I did an explanation at the main definition itself. Of course, I could copy-paste or move it up there if this is preferred

Reply:
Copying (perhaps a short of) the comment into the module docstring would indeed help, I think.

Reply:
Done